test/integration/start_stop_delete_test.go

```diff
@@ -103,8 +103,7 @@ func TestStartStop(t *testing.T) {
 						t.Fatalf("%s failed: %v", rr.Args, err)
 					}
 
-					names, err := PodWait(ctx, t, profile, "default", "integration-test=busybox", 2*time.Minute)
-					if err != nil {
+					if _, err := PodWait(ctx, t, profile, "default", "integration-test=busybox", 4*time.Minute); err != nil {
```

Comment:
the "names" was used by others, cant be deleted

Reply:
Thanks for finding this. It caused by a bad merge.